### PR TITLE
Create 10-26-2020.md

### PR DESCRIPTION
**Pull request by Cjbucker**

### CODE REVIEW

File: MeetingMinutes/Team/10-26-2020.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+team to the three others, it helped put into perspetive where we are. This will make moving forward easier, as we saw that other teams are in roughly
+the same spot as we are. 
+
+The team had also discussed reworking the meeting schedule we currently have. With a few of our memebers working thrid shift jobs, they struggle to meeting at
+regluar meeting hours. So we have to discuss the change to our scheduling to assure we keep up the communication.
+
+## Things Clarified
+
+We had decided that we would meet on Sunday (11/01) to talk more about scheduling and where we are headed for iteration one. As mentioned, this will
+be discussion around completly changing how our current meetings are scheduled. The discussion over iteration one will primarly be about who will be
```
Comment (AvidCuberCoding) [inline MeetingMinutes/Team/10-26-2020.md]: be a discussion around completely** and primarily**

File: MeetingMinutes/Team/10-26-2020.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+doing what work for the first iternation. Also, the team will need to decide how we will be documenting our work as we continue.
```
Comment (AvidCuberCoding) [inline MeetingMinutes/Team/10-26-2020.md]: iteration

**Comment by AvidCuberCoding** [comment]
other than those spelling issues, it all looks good to me!

**Review by Foster98 — APPROVED** [review]

**Comment by JulianEPrice** [comment]
Looks good!

**Review by JulianEPrice — APPROVED** [review]

**Review by AvidCuberCoding — APPROVED** [review]

**Pull request merged**